src/components/QuranReader/MulkLearningPlanBanner/index.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect, useRef, useState } from 'react';
```

Comment:
Remove `Mulk` from the naming so that we can extend it in the future to other Surahs.